### PR TITLE
wip: whats new for 5.3

### LINKED ISSUE #13254: Release blog post for 5.3
* What's new in v5.3
* Blog post

### PR DESCRIPTION
**Pull request by xlson**
What's new in 5.3, no images yet.

Todo:
- [x] gif for TV and Kiosk mode (@marefr)
- [x] Image for Postgres query builder

Closes #13254

### CODE REVIEW

File: docs/sources/guides/whats-new-in-v5-3.md [hunk 1]
```diff
@@ -12,6 +12,40 @@ weight = -9
...
+
+**gif here**
+
+## Notification reminders
+
+Do you use Grafana alerting and have some notifications that are more important than others? Then it's possible to set reminders so that you get alerted until the problem is fixed. This is done on the notification channel itself and will affect all alerts that use that channel. Read more about reminders [here](http://docs.grafana.org/alerting/notifications/#send-reminders)
+
+## Postgres query builder
+
+Grafana 5.3 comes with a new graphical query builder for Postgres. Bringing Postgres integration more in line with some the other datasources and making it easier for both advanced and beginners to work with timeseries in Postgres.
```
Comment (marefr) [inline docs/sources/guides/whats-new-in-v5-3.md]: more in line with some **of** the other

We should have a link here as well to the Postgres datasource documentation page.

File: docs/sources/guides/whats-new-in-v5-3.md [hunk 2]
```diff
@@ -12,6 +12,40 @@ weight = -9
 
 # What's New in Grafana v5.3
 
+Grafana v5.3 brings new features, many enhancements and bug fixes. This article will detail the major new features and enhancements.
+
+- [TV mode]({{< relref "#tv-and-kiosk-mode" >}}) is improved
+- [Alerting]({{< relref "#notification-reminders" >}}) with notification reminders
+- [Postgres query builder]({{< relref "#postgres-query-builder" >}}) it's finally here!
```
Comment (marefr) [inline docs/sources/guides/whats-new-in-v5-3.md]: Unsure about "It's finally here". How about

`[Postgres]({{< relref "#postgres" >}}) with a new graphical query builder`

File: docs/sources/guides/whats-new-in-v5-3.md [hunk 3]
```diff
@@ -12,6 +12,40 @@ weight = -9
...
+
+Do you use Grafana alerting and have some notifications that are more important than others? Then it's possible to set reminders so that you get alerted until the problem is fixed. This is done on the notification channel itself and will affect all alerts that use that channel. Read more about reminders [here](http://docs.grafana.org/alerting/notifications/#send-reminders)
+
+## Postgres query builder
+
+Grafana 5.3 comes with a new graphical query builder for Postgres. Bringing Postgres integration more in line with some the other datasources and making it easier for both advanced and beginners to work with timeseries in Postgres.
+
+{{< docs-imagebox img="/img/docs/v53/postgres_query_still.png" class="docs-image--no-shadow" animated-gif="/img/docs/v53/postgres_query.gif" >}}
+
+## Improved OAuth support for Gitlab
```
Comment (marefr) [inline docs/sources/guides/whats-new-in-v5-3.md]: Need to somehow describe that this is a new native implemetation for gitlab oauth which requires configuration changes.

A link to gitlab oauth documentation should be included.

File: docs/sources/guides/whats-new-in-v5-3.md (hunk 2 above)
Comment (xlson) [reply docs/sources/guides/whats-new-in-v5-3.md]: How about, "Postgres gets a new query builder!"

File: docs/sources/guides/whats-new-in-v5-3.md [hunk 4]
```diff
@@ -12,6 +12,40 @@ weight = -9
...
+We've improved the TV & kiosk mode to make it easier to use. There's now an icon in the top bar that will let you cycle through the different display modes.
+
+Clicks:
+
+1. (TV mode) Removes the sidebar and most of the buttons in the top bar
+2. (Kiosk mode) Removes the top bar completely so that only the dashboard itself is showing
+
+Hit the escape key to go back to the default mode.
+
+**gif here**
```
Comment (xlson) [inline docs/sources/guides/whats-new-in-v5-3.md]: @marefr any chance you can look into this?

**Comment by marefr** [comment]
Looking at the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md) makes me feel that we may include some more things in the What's new. We've added a few more things recently.
 
What do you think about including some short descriptions of these?
* Annotations: Enable template variables in tagged annotations queries #9735
* Templating: New templating variable type Text box that allows free text input #3173
* Alerting: Concurrent render limit for graphs used in notifications #13401

**Review by bergquist — APPROVED** [review]

**Comment by marefr** [comment]
Some changes left on Stackdriver then from @daniellee and then we can merge this

**Comment by daniellee** [comment]
So @sunker is rewriting the paragraph on Stackdriver now. We can merge it when he is done.

File: docs/sources/guides/whats-new-in-v5-3.md [hunk 5]
```diff
@@ -24,9 +24,17 @@ Grafana v5.3 brings new features, many enhancements and bug fixes. This article
 
 ## Google Stackdriver
 
-Grafana v5.3 ships with built-in support for [Google Stackdriver](https://cloud.google.com/stackdriver/) and enables you to visualize your Stackdriver metrics in Grafana.
+Grafana v5.3 ships with built-in support for [Google Stackdriver](https://cloud.google.com/stackdriver/) and enables you to visualize your Stackdriver metrics in Grafana. 
 
-The datasource is still in the `beta` phase, meaning it's currently in active development and is still missing a few important features (templating queries, support for metrics with distribution type).
+Getting started with the plugin is easy. Simply create a GCE Service account that has access to the Stackdriver api scope, download the JWT file from Google and upload it in the Stackdriver datasource config page in Grafana and you should have a secure server to server authentication setup. Like other core plugins, Stackdriver has built in support for alerting. It also comes with support for heatmaps and basic variables.
```
Comment (marefr) [inline docs/sources/guides/whats-new-in-v5-3.md]: s/server to server/server-to-server?

**Review by marefr — COMMENTED** [review]
Minor comment added. There's a lot of text - do we have any nice related Stackdriver image we can add which adds some more context to the text?

File: docs/sources/guides/whats-new-in-v5-3.md (hunk 5 above)
Comment (sunker) [reply docs/sources/guides/whats-new-in-v5-3.md]: server-to-server! Thanks

**Comment by daniellee** [comment]
Image added for Stackdriver:

stackdriver-with-heatmap

**Review by daniellee — APPROVED** [review]

**Review by bergquist — APPROVED** [review]

**Pull request merged**